deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Pgpool-II for PostgresSQL add-ons
+position: 3
+shortdesc: How to configure and use Pgpool-II for PostgresSQL add-ons
+tags:
+- addons
+keywords:
+- balancing
+- connection
+- limit
+- load
+- pgpool
+- pool
+- postgres
+- proxy
+- sql
+---
+
+Pgpool-II is a tool available on application instances to help you manage your connection pool and load balancing of your PostgreSQL add-ons.
+
+{{< alert "warning" "Warning:" >}}
+Pgpool-II is not available on Docker instances. If you want to use Pgpool-II, you can add it and configure it in your Dockerfile.
+{{< /alert >}}
+
+## What is Pgpool-II?
+
+{{< readfile "/content/partials/pgpool.md" >}}
+
+## Why use Pgpool-II?
+
+**Connection pooling**: Pgpool-II maintains established connections to the PostgreSQL servers, and reuses them whenever a new connection with the same properties (i.e. user name, database, protocol version, and other connection parameters if any) comes in. It reduces the connection overhead, and improves the system's overall throughput.
```

Comment:
I'm not sure how this renders in HTML, maybe it should be better to use regular `###` titles like we usually do.